vendor/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -235,11 +256,18 @@ int loggerGccParserCollectActions(
 {
   size_t i;
   /* Position of the last include path + 1 */
+  char full_prog_path [PATH_MAX+1];
+  char *path_ptr;
```

Comment:
Put the space after the asterisk not before.

Reply:
The rest of the file is written in a way that the pointer specifier ties to the type itself, not to the variable.